app/models/container.rb

```diff
@@ -18,9 +19,6 @@ class Container < ApplicationRecord
   has_many   :metric_rollups, :as => :resource
   has_many   :vim_performance_states, :as => :resource
 
-  # Needed for metrics
-  delegate   :my_zone, :to => :ext_management_system
```

Comment:
@zeari are you sure this is not used anywhere else in the metrics capture execution?